Source/ImPlotWidgets/Public/ImPlotWidget.h

```diff
@@ -46,9 +56,11 @@ class IMPLOTWIDGETS_API SImPlotWidget : public SCompoundWidget
 	FString Title;
 	FString XLabel;
 	FString YLabel;
+	EUnit XUnit;
```

Comment:
Again I can't comment on unmodified lines:
I would change the above `XValueArrays`/`YValueArrays` to be the cached best-fitted X/Y values

Reply:
If you want to go a step further:
You could change these` XValueArrays`/`YValuesArray`/`ShowLine` to be a struct like this
```cpp
struct FImPlotLineData
{
    bool ShowLine = true;
    TArray<float> XValues;
    TArray<float> YValues; // the best-fitted y values
};
```
and then store an `TArray<FImPlotLineData> Lines;` in the class